api/provider/aws/defaults.go

```diff
@@ -20,6 +21,7 @@ Write-Host Cluster name set as: $clusterName -foreground green
 
 [Environment]::SetEnvironmentVariable("ECS_CLUSTER", $clusterName, "Machine")
 [Environment]::SetEnvironmentVariable("ECS_ENABLE_TASK_IAM_ROLE", "false", "Machine")
+[Environment]::SetEnvironmentVariable("ECS_LOGLEVEL", "debug", "Machine")
```

Comment:
is this a typo? I would assume `ECS_LOG_LEVEL`

Reply:
No. Per the [docs](https://github.com/aws/amazon-ecs-agent/blob/master/README.md#environment-variables) under the Environment Variables section, it is `ECS_LOGLEVEL`